app/src/main/java/com/example/coolcalc/Calc.kt

```diff
@@ -1,14 +1,14 @@
 package com.example.coolcalc
 
 import android.widget.TextView
+import com.example.coolcalc.databinding.ActivityMainBinding
 
 class Calc {
-
-        var listObjDoubleCalc = mutableListOf<Double>()
-        var alreadyAresult = false
-        var result = 0.0
-        var stringOfNumbersForList =""
-
+    var activityMainBinding:ActivityMainBinding ?= null
+    var listObjDoubleCalc = mutableListOf<Double>()
+    var alreadyAresult = false
+    var result = 0.0
+    var stringOfNumbersForList =""

```

Comment:
```suggestion
    var stringOfNumbersForList = ""

```
Spacing (sure, call me desperate)

Reply:
haha nope its ok, this was prior bad behavior on my part.

Reply:
no time like the present!